rs/execution_environment/tests/hypervisor.rs

```diff
@@ -9691,7 +9691,7 @@ fn canister_balance_is_roughly(
     );
 }
 
-#[cfg(not(all(target_arch = "aarch64", target_vendor = "apple")))]
+#[cfg(all(target_arch = "x86_64", target_os = "linux"))]
```

Comment:
Should we maybe just disable `aarch64`? Currently, macOS Intel is also disabled.

Reply:
We want to run the test on the platforms where the new signal handler is available, which is x86 Linux only.